cmd/scheduler/main.go

```diff
@@ -58,9 +54,12 @@ type Database struct {
 }
 
 func main() {
-	var filename string
+	var filename, defaultLocation, pubsubTopic, jobPrefix string
 
 	flag.StringVar(&filename, "config", "", "The file path of the config file in yaml format.")
+	flag.StringVar(&defaultLocation, "location", "us-central1", "The location where the scheduler is deployed.")
+	flag.StringVar(&pubsubTopic, "topic", "cloud-spanner-scheduled-backups", "The PubSub topic where the scheduler sends to.")
+	flag.StringVar(&jobPrefix, "prefix", "spanner-backup", "The name prefix of a scheduler job.")
```

Comment:
Perhaps match the flag names to the variable names. `topic` and `prefix` feel a little too vague.